### DISCUSSION TITLE
AUR package

### DISCUSSION THREAD

**Discussion by pjeanjean** [tomasklaen/uosc#272] [General]
Hey,

I recently found this project, and I like it a lot.
This message is just to let you know that I made a basic PKGBUILD for Arch-based distributions and published it on the AUR : https://aur.archlinux.org/packages/mpv-uosc-git

If there is any issue with it, I don't mind deleting it, otherwise I'll be happy to keep maintaining it.

**Comment by christoph-heinrich** [comment]
The new tagline is "Feature-rich minimalist proximity-based UI for MPV player", but apparently that hasn't been updated yet in the project description.

The latest release is 4.0.1, not 3.1.2. It doesn't really matter since it's a git package, but you might as well update the version number while you're at it :wink:.

Other then that LGTM, but I haven't tried it.

---
@tomasklaen I noticed that the [User Scripts](https://github.com/mpv-player/mpv/wiki/User-Scripts) still had the old tagline and preview, so I updated it.

**Comment by pjeanjean** [comment]
Good points, these are fixed ;)
